lib/model.dart

```diff
@@ -48,6 +50,16 @@ class TestEntityPlain implements TestEntity {
 
   TestEntityPlain(this.id, this.tString, this.tInt, this.tLong, this.tDouble);
 
+  // TODO remove later; isar v0.4.0 needs a "public zero-arg constructor"
+  // NOTE: you also need to fix isar.g.dart after each generation to use this
```

Comment:
Should this be left as is? Would it be possible to change the other databases to use the zero arg constructor?

Reply:
that would have performance impacts (though minor) - it's better to construct the object completely with properties